### ISSUE TITLE
bug: sst bind does not pass through --flag arguments

### ISSUE THREAD

**Issue by mjperrone** [sst/sst#3118]
```bash
echo hi --there
hi --there
```

```bash
sst bind echo hi --there
hi
```

I would expect that any flag arguments after `bind` would be passed through to the command, but this example shows that this is not the case.

Looks like this is implemented around [here](https://github.com/serverless-stack/sst/blob/4d9c70f3b874fcf55de5ff9b0eade44e2f48fddf/packages/sst/src/cli/commands/bind.ts#L63), and may have to do with `yargs` eating the flag.

**Comment by jayair** [comment]
Replied on Discord.

**Comment by juandavidkincaid** [comment]
@jayair where you able to provide a workaround?, you mentioned discord, but maybe would be helpful if you can add a link here?

**Comment by mjperrone** [comment]
@juandavidkincaid here's the discord link: https://discord.com/channels/983865673656705025/1128799325498118206/1128799325498118206

TL;DR No ergonomic workaround, no one currently planning to work on it but there are some pointers to help someone who might want to.

**Comment by thdxr** [comment]
We are transition this repo to https://github.com/sst/v2 and are closing this issue for now. If it is still relevant please reopen it there - sorry for the trouble

**Comment by callumgare** [comment]
I found a workaround for this in case anyone (like me) still needs to support sst v2 and was hitting this issue.
The command `bash -c 'sst bind "$(printf "%q " "$@")"' -- echo hi --there` should produce the desired `hi --there` output :)
